orca-peering/src/main/kotlin/com/netflix/spinnaker/orca/peering/PeeringAgent.kt

```diff
@@ -149,6 +148,39 @@ class PeeringAgent(
     }
   }
 
+  /**
+   * Propagate deletes
+   * NOTE: ids of executions (both orchestrations and pipelines) that have been deleted are stored in the deleted_executions table
+   * the "id/primarykey" on that table is an auto-incrementing int, so we use that as a "cursor" to know what we've deleted and
+   * what still needs to be deleted.
+   * There is no harm (just some wasted RDS CPU) to "deleting" an execution that doesn't exist
+   */
+  private fun peerDeletedExecutions() {
+    val deletedExecutionIds = srcDB.getDeletedExecutions(deletedExecutionCursor)
```

Comment:
any concern about the growth of the `deleted_executions` table and how much data this call might end up returning with `deletedExecutionCursor = 0`?

Reply:
the `SqlExectionRepository` deletes rows older than 2days. But as you said, I will make it configurable